components/hashglobe/src/lib.rs

```diff
@@ -29,12 +29,14 @@ trait Recover<Q: ?Sized> {
 #[derive(Debug)]
 pub struct FailedAllocationError {
     reason: &'static str,
+    /// The size we are allocating, if needed.
+    allocating_size: Option<usize>,
```

Comment:
`allocation_size`?

Reply:
okok, will update the field name. Thanks.